package.json

```diff
@@ -135,7 +135,7 @@
     "style-loader": "^0.13.1",
     "supertest": "^2.0.0",
     "url-loader": "^0.5.7",
-    "vanilla-framework": "git+https://git@github.com/vanilla-framework/vanilla-framework.git#e3b7034a1caf39e4ed87b9481d5fe0e9913c2b0f",
+    "vanilla-framework": "^1.5.0",
```

Comment:
Oh wait - it looks like we use npm-shrinkwrap to solve this problem. That's fine, but it means you need to be committing an update to the `npm-shrinkwrap.json` file.

Reply:
@nottrobin I guess vanilla-framework didn't get into npm-shrinkwrap.json because it's a dev dependency (and dev dependencies are not locked in npm-shrinkwrap).

So if we want vanilla locked - and I agree it would be a good idea, we should move it to regular dependencies, so they will appear in npm-shrinkwrap.

Reply:
@bartaz yes I agree. Given that this site is never going to be an NPM package, the difference between dependencies and devdependencies is academic. So let's add it to regular deps.